src/main/java/WayofTime/bloodmagic/client/gui/GuiItemRoutingNode.java

```diff
@@ -20,17 +20,12 @@
 @SideOnly(Side.CLIENT)
 public class GuiItemRoutingNode extends GuiContainer
 {
-    private GuiButton downButton;
-    private GuiButton upButton;
-    private GuiButton northButton;
-    private GuiButton southButton;
-    private GuiButton westButton;
-    private GuiButton eastButton;
-    private GuiButton incrementButton;
-    private GuiButton decrementButton;
+    private GuiButton downButton, upButton, northButton, southButton, westButton, eastButton, incrementButton, decrementButton;
```

Comment:
Moving fields onto one line doesn't fit the formatting of anywhere else in the project.